images/capi/packer/ova/vsphere.json

```
@@ -0,0 +1,9 @@
+{
+    "vcenter_server":"",
+    "username":"",
+    "password":"",
+    "datastore":"",
+    "folder": "",
+    "cluster": "",
+    "network": ""
+}
```

```suggestion
   "insecure_connection": "false",
}
```
I would add this variable in here, as its basically an cluster-dependent flag, that needs to be set on most clusters

---

I'm fine either way. As I mentioned in an un-related comment, I'm considering removing the `esx.json` and would also remove this `vsphere.json` file as well. It encourages people to fill in the blanks and then you have a dirty repo when you build. These JSON files should not be modified, but then they also serve no purpose in their current form. I'll probably rename this as examples and refer to them as such instead.

---

It's why I add it in the documentation of the builder, I'm in favor to make them example files of what can pointed by the PACKER_VAR_FILES